src/frida/application.py

```diff
@@ -119,7 +120,13 @@ def store_target(option, opt_str, target_value, parser, target_type, *args, **kw
         self._exit_status = None
         self._console_state = ConsoleState.EMPTY
         self._quiet = False
-
+        self._logfile = options.logfile
+        if self._logfile is not None:
+            try:
+                self._f = open(self._logfile, 'w')
```

Comment:
I missed one thing, sorry, could we use `options.logfile` directly here in the `if` and `open`, and replace `_f` by `_logfile`? The logfile path isn't used again, so we don't have to store it on the object, and `_logfile` leaves us with a more descriptive name than `_f`.